src/test/java/core/basesyntax/service/impl/ReaderImplTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Reader;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ReaderImplTest {
+    private static final Reader reader = new ReaderImpl();
```

Comment:
```suggestion
    private static Reader reader = new ReaderImpl();
```
винеси ініціалізацію цієї змінної в метод BeforeClass. зроби це також і для інших класів